releasenotes/notes/deprecate-dd-tracer-configs-b05936f5ab56d128.yaml

```diff
@@ -0,0 +1,20 @@
+---
+features:
```

Comment:
This write up is great but probably a bit overkill (don't want to overwhelm users with release notes). I think we can do away with the `features` section and just stick to what you have already for the `upgrade`/`deprecations` sections